### ISSUE TITLE
Not working with RecyclerView

### ISSUE THREAD

**Issue by brotoo25** [nekocode/Emojix#5]
Can't find a way to make it working with my recyclerview implementation.

**Comment by nekocode** [comment]
@brotoo25  Did you create your item views programmatically?

**Comment by brotoo25** [comment]
I've created a RecyclerView.Adapter with my custom xml view binded to it.

**Comment by nekocode** [comment]
@brotoo25  Update your library to the new 1.2 version and try to use `Emojix.wrapView()` to wrap the views that were created in your RecyclerView.Adapter.

**Comment by brotoo25** [comment]
Still waiting for it to be available on Jitpack, I'll let you know as soon as I test it. Tks!

**Comment by nekocode** [comment]
@brotoo25 Yes, please feed back to me after you test it. Thank you very much!
